organizers.html

```diff
@@ -436,7 +434,44 @@ <h2 class="occupation">Web Chairs and Team</h2>
           </figcaption>
         </a>
       </figure>
-  
+  <figure class="u-block-hover">
+        <a href="sites.google.com/view/mahmudulhasanrobin">
+          <img src="images/Mahmudul_Hasan_Robin.png">
+          </img>
+          <figcaption>
+            <a href="sites.google.com/view/mahmudulhasanrobin">
+              <div class="name">
+                Mahmudul Hasan <span style="font-size: small">(MSc Student)</span>
+              </div>
+            </a>
+            <div class="background">
+              <div style="float:left;">
+                University of Texas Rio Grande Valley
+              </div>
+
+            </div>
+          </figcaption>
+        </a>
+      </figure>
+
+
+<!--   <figure class="u-block-hover">
```

Comment:
can we remove redundant code here?

Reply:
Also it would be great to use "github squash commits" to make these as single one commit, too keep a neat commit history.

Reply:
> can we remove redundant code here?

Yes, Please remove the redundant code here and merge it to master.

I will be waiting to see it live. Thanks.